core/src/main/java/io/honeycomb/opentelemetry/android/HoneycombOptions.kt

```diff
@@ -159,21 +159,17 @@ data class HoneycombOptions(
     val logsEndpoint: String,
     val sampleRate: Int,
     val debug: Boolean,
-
```

Comment:
well, it's mildly annoying that it removed the grouping. but not worth fighting the linter over